### PR TITLE
feat: add completion command

### PR DESCRIPTION
**Pull request by tisonkun**
Usage:

```
NAME:
   envd completion - Install shell completion scripts for envd

USAGE:
   envd completion [command options] [arguments...]

CATEGORY:
   Management Commands

OPTIONS:
   --shell value, -s value [ --shell value, -s value ]  Shell type to install completion
   --help, -h                                           show help (default: false)
```

### CODE REVIEW

File: pkg/app/completion.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/cockroachdb/errors"
+	"github.com/sirupsen/logrus"
+	ac "github.com/tensorchord/envd/pkg/autocomplete"
```
Comment (gaocegege) [inline pkg/app/completion.go:20]: Please move this import to a new group, to make linter happy.

**Review by gaocegege — DISMISSED** [review]
Thanks for your contribution! :tada: :+1:

**Comment by tisonkun** [comment]
@gaocegege 

```
#3 docker-image://docker.io/tensorchord/envd-sshd-from-scratch:v0.2.5-rc.2
#3 resolve docker.io/tensorchord/envd-sshd-from-scratch:v0.2.5-rc.2 0.2s done
#3 ERROR: docker.io/tensorchord/envd-sshd-from-scratch:v0.2.5-rc.2: not found
```

It seems the failure caused by unstable CI env?

**Comment by gaocegege** [comment]
The test failures are not from the PR, I am merging it.

Thanks for your contribution! :tada: :+1:

**Comment by gaocegege** [comment]
@all-contributors Please add @tisonkun for the code contribution.

**Pull request merged**